brigade/plugins/tasks/files/write.py

```diff
@@ -0,0 +1,48 @@
+import difflib
+import os
+
+from brigade.core.task import Result
+
+
+def read_file(file):
+    if not os.path.exists(file):
+        return []
+    with open(file, "r") as f:
+        return f.read().splitlines()
+
+
+def generate_diff(filename, content, append):
+    original = read_file(filename)
+    if append:
+        c = list(original)
+        c.extend(content.splitlines())
+        content = c
+    else:
+        content = content.splitlines()
+
+    diff = difflib.unified_diff(original, content, fromfile=filename, tofile="new")
+
+    return "\n".join(diff)
+
+
+def write(task, filename, content, append=False):
```

Comment:
I still think that `write` sounds very generic. Could it be `write_file` instead?

Reply:
I agree. Changing to `write_file`